src/core/memory.cpp

```diff
@@ -390,7 +395,7 @@ void MemorySystem::RasterizerMarkRegionCached(PAddr start, u32 size, bool cached
                     case PageType::RasterizerCachedMemory: {
                         page_type = PageType::Memory;
                         page_table->pointers[vaddr >> PAGE_BITS] =
-                            GetPointerForRasterizerCache(vaddr & ~PAGE_MASK);
+                            GetPointerForRasterizerCache(vaddr & ~PAGE_MASK) - (vaddr & ~PAGE_MASK);
```

Comment:
```(vaddr & ~PAGE_MASK) - (vaddr & ~PAGE_MASK);```???
Isn't this code contradict itself?

Is it supposed to be 
```suggestion
                            GetPointerForRasterizerCache(vaddr & ~PAGE_BITS) - (vaddr & ~PAGE_MASK);
```

or the other way round. I have no idea what is the correct code. Just comparing between the port.

Reply:
Nope, PAGE_BITS is not a bitmask.
Also, this is `GetPointerForRasterizerCache(vaddr & ~PAGE_MASK) - (vaddr & ~PAGE_MASK)`, the first one was a function call